mscore/plugin/api/qmlpluginapi.cpp

```diff
@@ -177,6 +177,19 @@ Element* PluginAPI::newElement(int elementType)
       return wrap(e, Ownership::PLUGIN);
       }
 
+//---------------------------------------------------------
+//   removeElement
+///   Disposes of an Element and its children.
+///   \param Element type.
+///   \since MuseScore 3.3
+//---------------------------------------------------------
+
+void PluginAPI::removeElement(Ms::PluginAPI::Element* wrapped)
+      {
+      Ms::Score* score = msc()->currentScore();
```

Comment:
It is better to use the wrapped element's score (`wrapped->element()->score()`) rather than the current score, the element does not necessarily have to belong to a current score.

Reply:
Makes sense.
Done.